kontent-ai/rich-text-resolver-js
# Extend default resolution

### Motivation

Fixes #20 #23 

Adds default resolution method for images, extends all default resolution methods to be optionally usable with Vue

### Checklist


### How to test

[See HTML transformer](https://github.com/kontent-ai/rich-text-resolver-js/blob/6b648dc6b7e7936b19a94bade5ecf6564e491b56/tests/transfomers/portable-text-transformer/html-transformer.spec.ts#L166) or [Vue transformer](https://github.com/kontent-ai/rich-text-resolver-js/blob/default-resolution/tests/transfomers/portable-text-transformer/vue-transformer.spec.ts) for example use. Documentation should be updated to reflect the universal capability of resolution methods (or they should be separated).
